### PR TITLE
Dev/virtual cam example

### PR DESCRIPTION
**Pull request by icywind**

### CODE REVIEW

**Comment by xiayangqun** [comment]
1. Can not open this new scene with unity 2018.4.0f1. You must make it can work with at last 2018.4.0f1
2. You should Submit PR to release/4.3 branch. Thanks

**Comment by icywind** [comment]
> 1. Can not open this new scene with unity 2018.4.0f1. You must make it can work with at last 2018.4.0f1
> 2. You should Submit PR to release/4.3 branch. Thanks

Done

File: API-Example-Unity/Assets/API-Example/Examples/Advanced/CustomCaptureAudio/CustomCaptureAudio.cs
Comment (xiayangqun) [inline API-Example-Unity/Assets/API-Example/Examples/Advanced/CustomCaptureAudio/CustomCaptureAudio.cs:1]: why this file had be changed?

File: API-Example-Unity/Assets/API-Example/Examples/Advanced/RenderWithYUV/RenderWithYUVSample.cs
Comment (xiayangqun) [inline API-Example-Unity/Assets/API-Example/Examples/Advanced/RenderWithYUV/RenderWithYUVSample.cs:1]: why this file had be changed?

**Comment by icywind** [comment]
Abandon and use a new branch

**Pull request closed without merging**